pubspec.lock

```diff
@@ -172,7 +172,7 @@ packages:
   delta_chat_core:
     dependency: "direct main"
     description:
-      path: "../flutter-deltachat-core"
+      path: "..\\flutter-deltachat-core"
```

Comment:
Am I right that these two backslashes are Windows specific? I remember that this very part has been changed automatically on my side when I built a new core the last time. 🤔 // @Boehrsi

Reply:
You are right. That is Windows specific :)

Reply:
Yep, this is annoying back and forth. Depending on the platform this is automatically changed over and over. The problem is, the file should be in the repo, so we can't exclude it. As it happens automatically right now it's not that bad and as soon as we have the plugin as Git / real dependency the problem should be fixed. For now we can just try to avoid committing it manually when checking before pushing stuff. Not ideal, but no nice idea on my side right how. I'm open for ideas ;).